Sources/DistributedActors/LifecycleMonitoring/LifecycleWatch.swift

```diff
@@ -26,7 +26,7 @@ public extension LifecycleWatch {
     @discardableResult
     func watchTermination<Watchee>(
         of watchee: Watchee,
-        @_inheritActorContext @_implicitSelfCapture whenTerminated: @escaping @Sendable(ActorSystem.ActorID) -> Void,
+        @_inheritActorContext @_implicitSelfCapture whenTerminated: @escaping @Sendable(ID) -> Void,
```

Comment:
oh that looks silly... formatters don't understand that all so well I guess;

Technically `@Sendable (ID) -> Void` and not `@Sendable(ID) -> Void` but I don't mind, let's get the bigger PR in and we can always reformat later :)